doc/benchmark.dtd

```diff
@@ -15,6 +15,7 @@
 
 <!ELEMENT option (#PCDATA)>
 <!ELEMENT propertyfile (#PCDATA)>
+<!ELEMENT multiproperty (#PCDATA)>
```

Comment:
Do we have another value for `kind` in order to allow explicitly setting the default value for users that want to document this?

Also this attribute needs to be specified in the DTD.